specification/sql/resource-manager/Microsoft.Sql/preview/2022-08-01-preview/examples/RefreshExternalGovernanceStatus.json

```diff
@@ -0,0 +1,29 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "sqlcrudtest-7398",
+    "serverName": "sqlcrudtest-4645",
+    "api-version": "2022-08-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "properties": {
+          "requestId": "00000000-0000-0000-0000-000000000000",
+          "requestType": "UpdatePurviewMetadata",
+          "queuedTime": "2/12/2022 8:33:27 PM",
+          "serverName": "testsvr.database.windows.net",
+          "status": "Completed"
+        },
```

Comment:
I dont see this response defined in your api call

Reply:
I dont fully understand. Whats your microsoft alias so that I may ping you there.

Reply:
Fixed this issue. The 200 response to reference the schema type.

Reply:
Resolved